thirdparty/dependencies.cmake

```diff
@@ -10,6 +10,7 @@ endif()
 
 set(_old_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
```

Comment:
looks like we need to store `_old_CMAKE_C_FLAGS` as well, because we change it later

Reply:
updated.